nflow-engine/src/main/java/io/nflow/engine/internal/dao/WorkflowInstanceDao.java

```diff
@@ -513,6 +509,11 @@ public WorkflowInstance getWorkflowInstance(int id, Set<WorkflowInstanceInclude>
     return instance;
   }
 
+  public DateTime getWorkflowInstanceStartTime(int id) {
+    return toDateTime(
+        jdbc.queryForObject("select min(execution_start) from nflow_workflow_action where workflow_id = ?", Timestamp.class, id));
```

Comment:
Could we store this data to nflow_workflow table instead?

Reply:
We could, see https://github.com/NitorCreations/nflow/pull/310